com.unity.postprocessing/PostProcessing/Runtime/PostProcessLayer.cs

```diff
@@ -1182,6 +1182,7 @@ int RenderBuiltins(PostProcessRenderContext context, bool isFinalPass, int relea
             context.uberSheet = uberSheet;
             context.autoExposureTexture = RuntimeUtilities.whiteTexture;
             context.bloomBufferNameID = -1;
+            context.currentEye = eye;
```

Comment:
For single-pass we can just set context.xrActiveEye = eye here instead of creating another eye variable (currentEye). Otherwise, there'll be 3 eye variables if we count the one in PostProcessLayer's for-eye loop. 
You can save and restore old value of xrActiveEye here if they are different from camera.